src/Context/ModalContext.js

```diff
@@ -0,0 +1,27 @@
+import React, { createContext, useContext, useState } from "react";
+
+const modalContext = createContext();
```

Comment:
put up default null if want nothing as it causes production issue

Reply:
Ok, will look into it.